.github/workflows/pythonlinters.yml

```diff
@@ -1,5 +1,5 @@
-# This workflow will install Python dependencies, run tests and lint with
-# a variety of Python versions For more information see:
+# This workflow will install Python 3.8 and other dependencies to lint the code.
+# For more information see:
```

Comment:
Should this still say `a variety of Python versions` instead of saying `3.8`?

Reply:
By comparing the matrix key with .github/workflows/pythontests.yml, I made the assumption that linting with only one version of Python is sufficient.

As Python 3.8 is currently used to lint, and as I do not know weather at least one of flake8, pyre, pylint or mypy requires Python 3.8 or not, I thought sticking to Python 3.8 was the safer choice: for now and in the near future (e.g. in case one of the aforementioned utilities has their codebase upgraded to Python 3.8).

Reply:
That makes perfect sense!